### ISSUE TITLE
Unable to deploy app using .net core 2.1.1 with buildpack 2.1.2

### ISSUE THREAD

**Issue by lunim** [cloudfoundry/dotnet-core-buildpack#189]
What version of Cloud Foundry and CF CLI are you using? (i.e. What is the output of running `cf curl /v2/info && cf version`?
cf version 6.37.0+a40009753.2018-05-25

What version of the buildpack you are using?
v 2.1.2

If you were attempting to accomplish a task, what was it you were attempting to do?
Deploy app successfully

I set the following in my csproj:

`<RuntimeFrameworkVersion>2.1.1</RuntimeFrameworkVersion>`

Logs:
```
Required dotnetframework versions: [2.1.1]
   -----> Installing dotnet-framework 2.1.1
          Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet-framework/dotnet-framework.2.1.1-rtm-26531-02.linux-amd64-93a2b619.tar.xz]
   -----> Publish dotnet
          Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
          Copyright (C) Microsoft Corporation. All rights reserved.

            Restoring packages for /tmp/app/<removed>.csproj...
            Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.1.1.
            Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.1.1.
            Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.1.1.
            Installing runtime.linux-x64.Microsoft.NETCore.App 2.1.1.
          /tmp/app/<removed> .csproj : warning NU1603: <removed>depends on Microsoft.AspNetCore.App (>= 2.1.1-rtm-30818) but Microsoft.AspNetCore.App 2.1.1-rtm-30818 was not found. An approximate best match of Microsoft.AspNetCore.App 2.1.1 was resolved.
            Restore completed in 4.58 sec for /tmp/app/<removed> .csproj.
          /tmp/app/<removed>.csproj : warning NU1603: <removed> depends on Microsoft.AspNetCore.App (>= 2.1.1-rtm-30818) but Microsoft.AspNetCore.App 2.1.1-rtm-30818 was not found. An approximate best match of Microsoft.AspNetCore.App 2.1.1 was resolved.
          /tmp/app/<removed>.csproj : error : The project was restored using Microsoft.AspNetCore.App version 2.1.1, but with current settings, version 2.1.1-rtm-30818 would be used instead.  To resolve this issue, make sure the same settings are used for restore and for subsequent operations such as build or publish.  Typically this issue can occur if the RuntimeIdentifier property is set during build or publish but not during restore.
          **ERROR** Unable to run dotnet publish: exit status 1
```

**Comment by ameyer-pivotal** [comment]
@lunim This is likely due to an ongoing issue with dotnet-core's build process (see https://github.com/dotnet/cli/issues/9456 for a similar circumstance). We're working with them to make our integrations play more nicely. For now, I'll patch this.

**Comment by ameyer-pivotal** [comment]
@lunim Feel free to give commit `cdb7f9be0b102253d623955af4efb7eba40fe35b` on the `develop` branch a try. Our integration tests are running, then we'll cut a release tomorrow if all goes well.

**Comment by tylerphelan** [comment]
@lunim This should be fixed in the latest buildpack release 2.1.3. Closing this issue, please re-open if you continue to see the same problems.

**Comment by lunim** [comment]
@tylerphelan I was able to push successfully with 2.1.3, thanks!
